model-server/src/test/kotlin/org/modelix/model/server/ModelClientTest.kt

```diff
@@ -104,6 +104,18 @@ class ModelClientTest {
         }
     }
 
+    @Test
+    fun `can retrieve server id initially`() = runTest {
+        val modelClient = createModelClient()
+        val serverId = modelClient.get("server-id")
+        assertNotNull(serverId)
```

Comment:
```suggestion
        val modelClient = createModelClient()
        
        val serverId = modelClient.get("server-id")
        
        assertNotNull(serverId)
```

For the arrange-act-assert structure?

Reply:
Done